lua/neotest-kotlin/init.lua

```diff
@@ -8,57 +8,54 @@ local filter = require("neotest-kotlin.src.filter")
 local treesitter_query = require("neotest-kotlin.kotest-treesitter-query")
 local output_parser = require("neotest-kotlin.src.output-parser")
 
-local adapter = { name = "neotest-kotest" }
+local M = {}
```

Comment:
I don't really understand the purpose of this useless wrapper, but it's just lifted directly from

https://github.com/nvim-neotest/neotest/blob/ef492755730e59e1d8122c461abbd086bee4c76b/lua/neotest/adapters/interface.lua